tests/testthat/test-metadata.R

```diff
@@ -165,7 +165,9 @@ test_that("xportr_df_label: Correctly applies label when data is piped", {
   df <- data.frame(x = "a", y = "b")
   df_meta <- data.frame(dataset = "df", label = "Label")
 
-  df_spec_labeled_df <- df %>% xportr_df_label(df_meta)
+  df_spec_labeled_df <- df %>%
+    xportr_df_label(df_meta) %>%
+    xportr_df_label(df_meta)
```

Comment:
how come there are two calls of the same function?

Reply:
The second pipe tests that `_xportr.df_arg_` is tested

Reply:
Yes, As Eli mentioned. It's testing two things here.

```
df_spec_labeled_df <- df %>%
    xportr_df_label(df_meta)
```
and
df_spec_labeled_df <- df_spec_labeled_df %>%
    xportr_df_label(df_meta)
```